packages/console/src/pages/EnterpriseSsoDetails/Experience/LogosUploader/index.tsx

```diff
@@ -29,9 +34,15 @@ function LogosUploader() {
             render={({ field: { onChange, value, name } }) => (
               <ImageUploader
                 name={name}
-                className={styles.frame}
+                className={conditional(
+                  isDarkModeEnabled && classNames(styles.frame, value && styles.frameBackground)
```

Comment:
We don't need to mix `conditional` with `classNames`, since `classNames` can also be used in a "conditional" way